wallet/txauthor/author_test.go

```diff
@@ -41,15 +43,24 @@ func makeInputSource(unspents []*wire.TxOut) InputSource {
 	// Return outputs in order.
 	currentTotal := dcrutil.Amount(0)
 	currentInputs := make([]*wire.TxIn, 0, len(unspents))
-	f := func(target dcrutil.Amount) (dcrutil.Amount, []*wire.TxIn, [][]byte, error) {
+	redeemScriptSizes := new([]int)
```

Comment:
don't use `new` here, just use `var` to create the zero value. `f` will close over it already.